src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -555,13 +556,21 @@ def _apply_cluster_overrides(
         return settings
 
     def _job_settings(self, step_name: str, remote_wheel: str):
+        def get_purge_time() -> str:
+            return (datetime.utcnow() + JOBS_PURGE_TIMEOUT).strftime("%Y%m%d%H")
```

Comment:
No minutes?

Reply:
Minutes are probably not required for our infra. Two extra characters saved in the name might be very helpful